### ISSUE TITLE
How to remove the seo alert in seo & url page

### ISSUE THREAD

**Issue by Maofree** [PrestaShop/PrestaShop#12047]
hi
how is it possible remove this alert in seo & url page?

**Screenshots**
![1](https://user-images.githubusercontent.com/653230/50769155-57872080-1283-11e9-8bef-26011e8d3213.png)

**Additionnal information**
PrestaShop version: 1.7.5.0
PHP version: 7.0

**Comment by khouloudbelguith** [comment]
Hi @maofree,

Thanks for your report.
It could be an improvement issue to remove the `seo-urls-showcase-card` definitely.
In our case PS1.7.5.0, if we remove it & we refresh the page => it is displayed again.
@marionf @colinegin what do you think?
Thanks!

**Comment by colinegin** [comment]
Hello,

It was not the expected behaviour indeed. If the merchant chooses to close the pop-up, it should not be displayed again. @PrestaShop/prestashop-core-developers @eternoendless  Do you think we could add this bug fix to the 1.7.5.1 roadmap ?

Thanks !

**Comment by eternoendless** [comment]
@colinegin Currently the "keep showcase card closed" functionality has not been implemented.

I started working on a simple implementation of it but then I realized:

1. As a merchant, if I close the card, it stays closed forever. What happens if I did it by mistake? Shouldn't I have an option somewhere to reset it?
2. On the simplest of implementations, closing it makes it closed for _all_ employees, not just me (same as the onboarding tutorial BTW).
